javascript/ql/src/semmle/javascript/InclusionTests.qll

```diff
@@ -71,8 +71,7 @@ module InclusionTest {
       count(this.getACallee()) = 1 and
       count(callee.getAReturnedExpr()) = 1 and
       not this.isImprecise() and
-      inner.getContainerNode().getALocalSource().getEnclosingExpr() = callee.getAParameter() and
-      inner.getContainedNode().getALocalSource().getEnclosingExpr() = callee.getAParameter()
+      inner.getContainerNode().getALocalSource() = DataFlow::parameterNode(callee.getAParameter())
```

Comment:
I do not understand this change. 
The previous version used both `getContainedNode` and `getContainerNode`.

Reply:
You are right, that was a mistake from my end, I didn't notice the `Contained`/`Container`. 

Everything ends up working anyway, as `getContainedNode` is called all the places the `InclusionTest` is used. And thus the `getContainedNode` constraint still enforced, just not in the CharPred.

I fixed it anyway (and refactored two other `getEnclosingExpr()`).